src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.request.CinemaHallRequestDto;
+import mate.academy.spring.dto.response.CinemaHallResponseDto;
+import mate.academy.spring.mapper.CinemaHallMapper;
+import mate.academy.spring.service.CinemaHallService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema_halls")
+public class CinemaHallController {
+    private final CinemaHallService cinemaHallService;
+    private final CinemaHallMapper cinemaHallMapper;
+
+    public CinemaHallController(CinemaHallService cinemaHallService,
+                                CinemaHallMapper cinemaHallMapper) {
+        this.cinemaHallService = cinemaHallService;
+        this.cinemaHallMapper = cinemaHallMapper;
+    }
+
+    @GetMapping
+    public List<CinemaHallResponseDto> getAllCinemaHalls() {
```

Comment:
```suggestion
    public List<CinemaHallResponseDto> getAll() {
```
doesn't make sence to add entity name, taking into account the name for the controller, it's understandable what you're trying to get